<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>URL地址:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="siteurl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>配置:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="config">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>691</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>备注:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="remarks">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>691</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="type_id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="script">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="coding">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>250</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>记录最后配置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sitepass">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="pass_checkBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>显示密码</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>siteurl</tabstop>
  <tabstop>sitepass</tabstop>
  <tabstop>config</tabstop>
  <tabstop>remarks</tabstop>
  <tabstop>type_id</tabstop>
  <tabstop>script</tabstop>
  <tabstop>coding</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
